kernel/src/lib.rs

```diff
@@ -99,6 +99,7 @@ pub mod table_configuration;
 pub mod table_features;
 pub mod table_properties;
 pub mod transaction;
+pub mod transforms;
```

Comment:
Since there's nothing public, I think we can just make this pub(crate). If things break, just keep it pub mod 
```suggestion
pub(crate) mod transforms;
```